common/config/babel/.babelrc.js

```diff
@@ -24,6 +24,8 @@ process.env['COMMUNICATION_REACT_FLAVOR'] === 'stable' &&
         'demo',
         // dialpad
         'dialpad',
+        // Data loss prevention feature
+        'dlp',
```

Comment:
maybe use the full name like "data-loss-prevention"? It would be easier for someone who never knows this to understand the logic when they see conditional compilation